### PR TITLE
Improvement/aave v2 v3 tests

### PR DESCRIPTION
**Pull request by rajko-z**

### CODE REVIEW

File: addresses/mainnet.json [hunk 1]
```diff
@@ -1213,6 +1213,17 @@
         "registryIds": [],
         "history": []
     },
+    {
+        "name": "AaveClaimStkAave",
+        "address": "0x5Dcd9Dc0185a6Ab07a31e5284D16Ce9f0A79Ce99",
+        "id": "0xd93d7e7f",
+        "path": "contracts/actions/aave/AaveClaimStkAave.sol",
+        "version": "1.0.0",
+        "inRegistry": true,
```
Comment (yippee-ki-yay) [inline addresses/mainnet.json:1222]: It's not in registry, set to false, and changeTime to 0

File: test/aaveV3/aave-tests.js [hunk 2]
```diff
@@ -20,25 +20,23 @@ const {
...
-        const network = hre.network.config.name;
 
         before(async () => {
+            console.log('NETWORK:', network);
             senderAcc = (await hre.ethers.getSigners())[0];
             proxy = await getProxy(senderAcc.address);
-            const aaveMarketContract = await hre.ethers.getContractAt('IPoolAddressesProvider', AAVE_MARKET_OPTIMISM);
+
+            const aaveMarketContract = await hre.ethers.getContractAt('IPoolAddressesProvider', addrs[network].AAVE_MARKET);
             const poolAddres = await aaveMarketContract.getPool();
```
Comment (yippee-ki-yay) [inline test/aaveV3/aave-tests.js:35]: poolAddress*

File: test/aave/aave-tests.js [hunk 3]
```diff
@@ -437,47 +482,39 @@ const aaveClaimStkAaveTest = async () => {
         this.timeout(150000);
 
         const stkAaveAddr = '0x4da27a545c0c5B758a6BA100e3a049001de870f5';
-
+        const OWNER_ACC = '0xe97f5363b77424332aae94e358a41ccbeb9e454b';
```
Comment (yippee-ki-yay) [inline test/aave/aave-tests.js:485]: Would change to `USER_ACC` `CLAIM_ACC` as we use OWNER_ACC already in utils.js with a diff. meaning

**Review by yippee-ki-yay — APPROVED** [review]
Good cleanup, tests are passing, few small details to fix but overall looking good

File: addresses/mainnet.json (hunk 1 above)
Comment (rajko-z) [reply addresses/mainnet.json]: Fixed

File: test/aaveV3/aave-tests.js (hunk 2 above)
Comment (rajko-z) [reply test/aaveV3/aave-tests.js]: Fixed

File: test/aave/aave-tests.js (hunk 3 above)
Comment (rajko-z) [reply test/aave/aave-tests.js]: Fixed

**Pull request merged**